dashboard/test/ui/runner.rb

```diff
@@ -66,9 +66,12 @@ def main(options)
   run_results = Parallel.map(browser_feature_generator, parallel_config(options.parallel_limit)) do |browser, feature|
     run_feature browser, feature, options
   rescue => exception
+    Infrastructure::Logger.flush
     ChatClient.log "Exception: #{exception.message}", color: 'red'
     raise
   end
+  # Flush metrics about feature runs into cloudwatch
+  Infrastructure::Logger.flush
```

Comment:
What about something like this?

```suggestion
  rescue => exception
    ChatClient.log "Exception: #{exception.message}", color: 'red'
    raise
  ensure
    # Flush metrics about feature runs into cloudwatch
    Infrastructure::Logger.flush
  end
```